_pytest/doctest.py

```diff
@@ -291,6 +295,18 @@ def _get_allow_bytes_flag():
     return doctest.register_optionflag('ALLOW_BYTES')
 
 
+def _get_report_choices():
+    import doctest
+    return dict(
+        zip(
+            _get_report_choices_keys(),
+            (doctest.REPORT_UDIFF, doctest.REPORT_CDIFF, doctest.REPORT_NDIFF, doctest.REPORT_ONLY_FIRST_FAILURE, 0, )
+        )
+    )
+
+def _get_report_choices_keys():
```

Comment:
Nitpick here: wouldn't be simpler to just have `_get_report_choices()` and use `_get_report_choices().keys()` when you need the keys instead?